src/pages/rooms/[id]/questions/form.tsx

```diff
@@ -0,0 +1,182 @@
+  const roomId = router.query.id;
+  const answersFormatSelect = (e: any) => {
+    setAnswersFormat(e.target.value);
+  };
+
+  const onChangeFixAnswer = (newValue: string, index: number) => {
+    setFixAnswers((currentFixAnswers) => {
+      const copyFixAnswers = [...currentFixAnswers];
+      copyFixAnswers[index] = newValue;
+      return copyFixAnswers;
+    });
+  };
+
+  const onAddFixAnswer = (e: SyntheticEvent) => {
+    e.preventDefault();
+    setFixAnswers((currentFixAnswers) => [...currentFixAnswers, '']);
+  };
+
+  const onRemoveFixAnswer = (e: SyntheticEvent) => {
+    e.preventDefault();
+    setFixAnswers((currentFixAnswers) => {
+      const copyFixAnswers = [...currentFixAnswers];
+      if (copyFixAnswers.length > 1) {
+        currentFixAnswers.pop();
+      }
+      return copyFixAnswers;
+    });
+  };
+
+  const question = (): QuestionCreate => {
+    return {
+      text,
+      answersFormat,
+      fixAnswers: answersFormat === 'fix' && fixAnswers,
+    };
+  };
+
+  const onSubmit = (e: SyntheticEvent) => {
+    e.preventDefault();
+    api<Question>(
+      'post',
+      `api/rooms/${roomId}/questions`,
+      (question) => {
+        router.push(`/rooms/${roomId}/questions/${question.id}`);
+      },
+      question(),
+    );
+  };
+
+  return (
+    <>
+      <h2>Create poll</h2>
+      <p>
+        Please select a response type for your poll: <b>fix answers</b>,{' '}
+        <b>free answers</b> or <b>temperature check</b>.
+      </p>
+      <ul>
+        <li>
+          <b>Fix answers:</b> You create a set of answers and users select one
+          of them. After choosing an answer users see the relative amount of
+          choices for each answer. Maximun number of answers: 50.
+        </li>
+        <li>
+          <b>Free answers:</b> Users insert their answers in an empty text
+          field. Each answer is displayed in a text block. Maximum number of
+          characters: 500.
+        </li>
+        <li>
+          Temperature check: You create a question to which the answer can be
+          given on a scale from 1 to 10. Users can only select one number. Every
+          response is visualised in a histogram.
+        </li>
+      </ul>
+      <form className="ui form" onSubmit={onSubmit}>
+        <div className="field">
+          <label>
+            <h3>Your polling question</h3>
+          </label>
+          <input
+            type="text"
+            value={text}
+            placeholder="Type your question"
+            onChange={(e) => {
+              setText(e.target.value);
+            }}
+          />
+        </div>
+        <div className="field">
+          <label>
+            <h3>Your answer type</h3>
+          </label>
+          <input
+            type="radio"
+            value="fix"
+            checked={answersFormat === 'fix'}
+            onChange={answersFormatSelect}
+            name="typeOfAnswer"
+          />{' '}
+          fix answers
```

Comment:
```suggestion
          />
          {' '} fix answers
```
child text nodes on same line is more readable at this text size